apps/sms/test/marionette/lib/views/inbox/view.js

```diff
@@ -17,4 +17,12 @@ InboxView.prototype.goToFirstThread = function() {
   return conversation;
 };
 
+InboxView.prototype.createNewMessage = function() {
+  this.accessors.createNewMessageButton.tap();
+  var ComposerView = require('../composer/view');
+  var composer = new ComposerView(this.client);
+  composer.accessors.waitToAppear();
```

Comment:
Just wondering, do you still think that ```waitToAppear``` is not view method, but rather ```accessors```'s one? It feels weird to use other view's accessors here, but won't block on this.